src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int vaultOfClasses = 5;
```

Comment:
I think that is a constant.
Make it constant and think about [name](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names).